ajax/libs/jquery.formstyler/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "name": "jquery.formstyler",
```

Comment:
Let's use git repo name `jQueryFormStyler` as this library's name. Sorry for not noticing this previously.